app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -196,9 +202,16 @@
       nextLevelVideo: ->
         if me.isStudent() and @course._id == utils.courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE and !me.showHeroAndInventoryModalsToStudents()
           return utils.videoLevels[@nextLevel.original]
+      inExam: ->
+        exam = storage.load('exam', true)
+        return _.any exam.problems, (courseLevels) =>
+          return _.find(courseLevels.levels, {slug: @level.slug})
+
     }
     methods: {
       marked
+      onReturnExam: ->
+        window.close()
```

Comment:
we could actually open the exam URL of progress, right? If it's possible, lets' do that.

Reply:
we can while, i don't want to open too many pages. or you think we can open the level link in current tab and make this button return to progress page?